test/test_extended_models.py

```diff
@@ -335,3 +335,16 @@ def builder(*, weights=None, flag):
 
         with pytest.raises(ValueError, match="weights"):
             builder(pretrained=True, flag=False)
+
+    @pytest.mark.parametrize(
+        "model_fn",
+        TM.list_model_fns(models)
+        + TM.list_model_fns(models.detection)
+        + TM.list_model_fns(models.quantization)
+        + TM.list_model_fns(models.segmentation)
+        + TM.list_model_fns(models.video)
+        + TM.list_model_fns(models.optical_flow),
+    )
+    def test_all_models(self, model_fn):
+        with pytest.warns(UserWarning, match="deprecated"):
+            model_fn(pretrained=True)
```

Comment:
This will slowdown the tests. I can remove it but it can be used to avoid missing it on the future.

Reply:
Good question, it goes from [2:40](https://app.circleci.com/pipelines/github/pytorch/vision/20041/workflows/2e3bb4bc-ebe2-4a16-9fbb-72047860ba00/jobs/1628137) to [5:03](https://app.circleci.com/pipelines/github/pytorch/vision/20056/workflows/dc139173-7870-439f-8cb1-c3860e119759/jobs/1629547).

Reply:
Sounds good, I just added the `@run_if_test_with_extended` decorator to run only on the `unittest_linux_cpu` job.